misk/src/main/kotlin/misk/metrics/MetricsConfig.kt

```diff
@@ -0,0 +1,7 @@
+package misk.metrics
+
+import misk.config.Config
+import misk.metrics.backends.MetricsBackendConfig
+
+data class MetricsConfig(val backends: MetricsBackendConfig) : Config
```

Comment:
This isn't used right now, should config be set up in a different PR?

Reply:
The top-level config is the top-level container for the back-end specific configs introduced in this PR.  IMO it makes the most sense to add all of that configuration in this PR.  We don't use the metrics config in exemplar because I don't think we want to have an exemplar configuration specifying a specific metrics backend, with associated port, project, etc.